actionsystemtest/lib/action_system_test.rb

```diff
@@ -0,0 +1,67 @@
+# System tests are similar to Integration tests in that they incorporate multiple
+# controllers and actions, but can be used to simulate a real user experience.
+# System tests are also known as Acceptance tests.
+#
+# To create a System Test in your application, extend your test class from
+# <tt>ActionSystemTestCase</tt>. System tests use Capybara as a base and
+# allow you to configure the driver. The default driver is
+# <tt>RailsSeleniumDriver</tt> which provides a Capybara and the Selenium
+# Driver with no configuration. It's intended to work out of the box.
```

Comment:
I think we should downplay the `RailsSeleniumDriver` stuff. People shouldn't need to know the internals of this when first learning about it. `RailsSeleniumDriver` feels like essentially a private class that's just used below a set of configuration points.